superset/viz.py

```diff
@@ -518,6 +522,19 @@ def data(self):
 
     def get_csv(self):
         df = self.get_df()
+        if df is None:
```

Comment:
can we comment when we will hit this if block?

Reply:
When we export CSV for filter box (the charts which don't have any query obj). So, this avoids the error when the query's empty.